packages/carbon-react/.storybook/main.js

```diff
@@ -7,9 +7,48 @@
 
 'use strict';
 
+const fs = require('fs');
+const glob = require('fast-glob');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const path = require('path');
 
+const stories = glob
+  .sync(
+    [
+      './Welcome/Welcome.stories.js',
+      '../src/**/*.stories.js',
+      '../src/**/*.stories.mdx',
+      '../../react/src/**/next/*.stories.js',
+      '../../react/src/**/next/*.stories.mdx',
+      '../../react/src/**/*-story.js',
+    ],
+    {
+      cwd: __dirname,
+    }
+  )
+  .filter((match) => {
```

Comment:
Could you add a comment/note on what this filter is trying to do with the matches stories? I think this will help out down the line if we revisit this or are running into some kind of problem with this chunk of code.